definitions/sonarqube-common.d.ts

```diff
@@ -28,7 +28,7 @@ declare module 'sonarqube-common/sonarqube-common' {
     export function applySonarQubeAnalysisParams(toolRunner:ToolRunner, projectName?, projectKey?, projectVersion?):ToolRunner;
 
     // Fetches configured SonarQube endpoint details.
-    export function getSonarQubeEndpointFromInput(inputFieldName):SonarQubeEndpoint;
+    export function getSonarQubeEndpoint():SonarQubeEndpoint;
```

Comment:
Reducing arguments to gather them up inside sonarqube-common rather than inside the individual sonar__.ts files.